src/index.html

```diff
@@ -22,6 +25,395 @@
+          src="images/logo.png"
+          alt="Moyo logo"
+        />
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__nav_list">
+          <li class="header__nav_item">
+            <a
+              class="header__nav_link is-active"
+              href="#Apple"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="header__nav_item">
+            <a
+              class="header__nav_link"
+              href="#Samsung"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="header__nav_item">
+            <a
+              class="header__nav_link"
+              href="#Smartphones"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="header__nav_item">
+            <a
+              data-qa="nav-hover"
+              class="header__nav_link"
+              href="#Laptops&Computers"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="header__nav_item">
+            <a
+              class="header__nav_link"
+              href="#Gadgets"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="header__nav_item">
+            <a
+              class="header__nav_link"
+              href="#Tablets"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="header__nav_item">
+            <a
+              class="header__nav_link"
+              href="#Photo"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="header__nav_item">
+            <a
+              class="header__nav_link"
+              href="#Video"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="cards">
+      <div
+        class="cards__card"
+        data-qa="card"
+      >
+        <img
+          class="cards__card-image"
+          src="./images/imac.jpeg"
+          alt="imac img"
+        />
+
+        <h2 class="cards__card-title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="cards__card-info">Product code: 195434</p>
+
+        <div class="cards__card-rate">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <span class="cards__card-revie">Reviews: 5</span>
+        </div>
+
+        <div class="cards__card-price">
+          <span class="cards__card-price-text">Price:</span>
+          <span class="cards__card-price-amount">$2,199</span>
+        </div>
+
+        <a
+          href="#"
+          class="cards__card-button"
+          data-qa="card-hover"
```

Comment:
The `data-qa` attribute is used for testing purposes and should not be present in the production code unless specifically required for automated tests.